assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java

```diff
@@ -384,6 +384,15 @@
      */
     public Collection<Assignment> getAssignmentsForContext(String context);
 
+    /**
+     * Access all the Assignments that are deleted
+     *
+     * @param context -
+     *                Describes the portlet context - generated with DefaultId.getChannel().
+     * @return List All the deleted assignments will be listed
+     */
+    public Collection<Assignment> getDeletedAssignmentsForContext(String context);
```

Comment:
Lets add a unit test for this method...

Reply:
You're right @ern, I've added a unit test, it deletes an assignment and checks the returned Collection.